components/events/AccordionSection.vue

```diff
@@ -0,0 +1,67 @@
+<template>
+  <section
+    class="accordion__section"
+  >
+    <h2 class="copy__title">
+      {{ document.title }}
+    </h2>
+    <cv-accordion :align="`end`">
+      <nuxt-content :document="document" />
+    </cv-accordion>
+  </section>
+</template>
+
+<script lang="ts">
+import Vue from 'vue'
+import { Component, Prop, Watch } from 'vue-property-decorator'
+import { IContentDocument } from '@nuxt/content/types/content'
+
+@Component
+export default class AccordionSection extends Vue {
+  @Prop({ type: String }) title: string | undefined
+  @Prop({ type: Object }) document!: IContentDocument
+
+  cvTemplate (slot: string, children: any[]) {
+    return {
+      type: 'element',
+      tag: 'template',
+      props: { [slot]: '' },
+      children
+    }
+  }
+
+  cvAccordionItem (element: any) {
+    return {
+      type: 'element',
+      tag: 'cv-accordion-item',
+      props: {},
+      children: [this.cvTemplate('title', [element]), this.cvTemplate('content', [])]
+    }
+  }
+
+  @Watch('document', { immediate: true })
+  onDocumentChange (val: IContentDocument) {
+    val.body.children
+      .filter((element:any) => element.tag === 'item')
+      .forEach((element:any) => {
```

Comment:
any types are frown upon 😨

Reply:
The definition of IContentDocument is very bad: 
```
export interface IContentDocument extends Record<string, any> {
  dir: string;
  path: string;
  extension: '.md' | '.json' | '.yaml' | '.xml' | '.csv' | string;
  slug: string;
  createdAt: Date;
  updatedAt: Date;
}
```
I can create some interfaces with reverse engineering to match the current model.